### DISCUSSION TITLE
I am a beginner in programming, may I know what is the problem with the code?

### DISCUSSION THREAD

**Discussion by Adedsw** [community/community#134751] [Programming Help]
### Select Topic Area

Question

### Body

![Screenshot (82)](https://github.com/user-attachments/assets/9a5d29ca-c358-45b5-98c6-e279045f11c4)
![Screenshot (83)](https://github.com/user-attachments/assets/bdfb94ac-e5ff-4f61-971a-de270aa7a879)
![Screenshot (84)](https://github.com/user-attachments/assets/9c1d5cf5-d72a-4945-ba95-fc5e52f6aa9a)

**Comment by Mahd25** [comment]
## Hello @Adedsw

Your problem can be solved by placing parentheses in front of the (lower method).

**example:**

`input("your content: ").lower()`

---
***tip:***
It is better to write the (import random) at the beginning of the file.

Also, if you use (") instead of (""") it is more suitable.

### I hope it was useful for you.

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @Adedsw ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Comment by geniuszly** [comment]
Hi! Python is very particular about indentation. Make sure that all code blocks (such as the ones inside `if` statements) are properly indented. From what I can see in your code, the indentation seems to be correct, but you need to ensure this is consistent throughout the entire program.
The program checks if the input is `help` using `.lower() `but doesn't check if it's an empty string or any other invalid entry. You might want to add a check to ensure the user only enters valid choices like `rock`, `paper`, or `scissors` after the game starts.
There’s a small typo in the instructions: `rock smahse scissors` should be `rock smashes scissors.`
The flow of the game should be as follows: display the welcome message, take the user’s choice, and compare it with the computer’s randomly generated choice to determine the winner.
Ensure that the ASCII art for rock, paper, and scissors is properly aligned when printed to the console. Sometimes the alignment might get messed up due to different text editors or terminal widths.
You have a check to see if the `user_choice` is valid. If the user enters something other than `rock`, `paper`, or `scissors`, they are asked to re-run the program. This is a good check, but instead of asking them to re-run, you could prompt them again within the same program.
The code checks for what the user and the computer chose, but it seems to print out the choice in a way that might be difficult to read. Ensure that the printed statements are clear, and the user knows exactly what they and the computer chose.
Make sure that the `random` module is correctly imported at the beginning of your code with `import random`.
